starboard/android/apk/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,5 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-7.2-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-7.3.3-all.zip
```

Comment:
This is pointing at `7.3.3` while above is at `7.2.2` - which one is correct and do they need to be aligned ?

Reply:
Check the versions here https://developer.android.com/build/releases/past-releases/agp-7-2-0-release-notes#compatibility-7-2-0.